app/themes/federal.tsx

```diff
@@ -614,6 +614,24 @@ theme.components = {
     },
   },
   MuiSelect: {
+    defaultProps: {
+      IconComponent: (props) => (
+        <span
+          {...props}
+          style={{
+            ...props.style,
+            right: 12,
+            transition: "transform 0.1s",
+          }}
+        >
+          <Icon
+            name="chevronDown"
+            size={16}
+            style={{ transform: "scale(1.5)" }}
+          />
+        </span>
+      ),
+    },
```

Comment:
I checked it at https://visualization-tool-git-feat-most-recent-x-axis-ixt1.vercel.app/storybook/index.html?path=/story/design-system-theme--components and it seems a bit weird ?  In the app it is good 🤔

Reply:
Not super sure why it's like that, I assume it's due to a fact that we render select element in a custom way – and generally we don't use native MUI elements, but have our own wrappers in the `Form` file. Maybe we could have a note somewhere to use "our" components, and not native ones? I tried to fix this problem, but then there are some padding issues when select element is expanded 🤔